eks/jx-requirements-eks.yml

```diff
@@ -0,0 +1,48 @@
+autoUpdate:
```

Comment:
what's about renaming this to _jx-requirements-eks.yml _? I kind of like the idea that applying the scripts create a canonical requirements file.

Reply:
that said, this seems to be the rendered requirements which probably should go into .gitignore and should not be checked in

Reply:
Ah, this file should've never made it into the PR.

This is the one generated by the script after you run it, there's a template somewhere else.